crates/gfx-plugin/src/lib.rs

```diff
@@ -50,7 +51,12 @@ use thiserror::Error;
 /// Builds a [`GraphicalApplication`].
 #[derive(Debug, Default)]
 pub struct GraphicalApplicationBuilder<AppBuilder> {
-    builder: AppBuilder,
+    app_builder: AppBuilder,
+    camera_movement_speed: Option<f32>,
+    camera_mouse_sensitivity: Option<f32>,
+    far_clipping_plane: Option<f32>,
+    near_clipping_plane: Option<f32>,
+    field_of_view: Option<Deg<f32>>,
```

Comment:
I assume these can/will be moved to the `Camera` Component when/if we implement the input system.